tic_tac_toe/game.py

```diff
@@ -9,7 +11,9 @@ def _position_is_empty_in_board(position, board):
 
     Returns True if given position is empty, False otherwise.
     """
-    pass
+    if board[position[0]][position[1]]=='-':
```

Comment:
This can be simplified by just using `return board[position[0]][position[1]] == '-'` as it will return `True` if it's empty, and `False` if it's not.